build.gradle

```diff
@@ -183,8 +183,8 @@ dependencies {
 
   compile group: 'uk.gov.hmcts.reform', name: 'java-logging', version: versions.reformLogging
   compile group: 'uk.gov.hmcts.reform', name: 'java-logging-appinsights', version: versions.reformLogging
-  compile group: 'uk.gov.hmcts.reform', name: 'sscs-common', version: '0.0.156'
-  compile group: 'uk.gov.hmcts.reform', name: 'sscs-pdf-email-common', version: '0.1.1'
+  compile group: 'uk.gov.hmcts.reform', name: 'sscs-common', version: '0.0.155-RC7'
+  compile group: 'uk.gov.hmcts.reform', name: 'sscs-pdf-email-common', version: '0.0.112'
```

Comment:
this version might have an issue with reform logging being 5.0.1

Reply:
What one should it be @ryandac ?

Reply:
3.0.1